### ISSUE TITLE
Simple example on webpack 2 does not work

### ISSUE THREAD

**Issue by nikoskalogridis** [rxaviers/globalize-webpack-plugin#38]
I am trying to use this plugin with webpack 2 and it seems that when producing a production build the produced i18n/*.js files have the same language (the development locale defined in the plugin options). I have made a simple repo for reproducing the issue https://github.com/nikoskalogridis/globalize-test

What am I doing wrong?

**Comment by nikoskalogridis** [comment]
It appears that for webpack 2 the https://github.com/rxaviers/globalize-webpack-plugin/blob/master/ProductionModePlugin.js#L229 regex is not correct.
in webpack 2 the produced file starts with
```
/***/ (function(module, exports, __webpack_require__) {
```
instead of the 
```
/***/ function(module, exports, __webpack_require__) {
``` 
that the plugin searches so it never matches

@rxaviers should I make a PR with this fix? Do we need to keep webpack v1 support?

**Comment by rxaviers** [comment]
Hi @nikoskalogridis thanks for your detailed issue! Sorry I couldn't go over it just yet, but your findings make sense to me. Yeap you can go ahead and submit a PR.  Let's try keeping support for both simultaneously. If it eventually becomes too difficult we can branch it into two versions, one to support webpack1, another to support webpack2.

Thanks so far

**Comment by rxaviers** [comment]
Note to self:
- [ ] When this is fixed, test #39.

**Comment by cahuja** [comment]
Are you expecting a PR for this anytime soon? Alternatively, I can submit one.

**Comment by rxaviers** [comment]
Hi @cahuja, it would be great if you could submit a PR. Thanks

**Comment by cahuja** [comment]
I submitted the following PR - https://github.com/rxaviers/globalize-webpack-plugin/pull/45 
Please let me know if you need anything else from me.

**Comment by rxaviers** [comment]
(Should be) Fixed by #45 (`v0.3.11`), if it's not please post comments and I will reopen. PS: This repo urgently needs tests.
